src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,100 @@
+@import '../utils/variables';
+
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect scales the card to 1.2 times its size, which might cause layout shifts. Consider using a more subtle scale value to prevent disrupting the layout.